build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+usage() { echo "Usage: ./build.sh [target_image_path]"; exit 1; }
```

Comment:
an example run of this script would be
 ```./build.sh gcr.io/google_appengine/golang-docker```
currently build.sh exists mostly for convenience (i.e. running locally) just as a wrapper around a cloud build call.